### ISSUE TITLE
Noise profile Fuji X-H1

### ISSUE THREAD

**Issue by schleiftier** [darktable-org/darktable#4281]
[darktable-noiseprofile-20200213.tar.gz](https://github.com/darktable-org/darktable/files/4195568/darktable-noiseprofile-20200213.tar.gz)

Here is a noise profile for the Fuji X-H1 that i generated following the instructions at https://pixls.us/articles/how-to-create-camera-noise-profiles-for-darktable/. 

From my testing it seems to work quite nicely.

Please include it in darktable!

Thanks & best regards,
Frederic

**Comment by rawfiner** [comment]
Hi @schleiftier 
Thanks for the profile, but we already have a noise profile for Fuji X-H1 (since 02/01/2020 actually, so it is quite recent and is not in dt 3.0.0 but will be in 3.0.1)

**Comment by johnny-bit** [comment]
I wanted to use this issue and ask - would there be a difference/noticeable difference between generated noise profiles based on older vs newer instructions from Andreas' article? And whether new noise profile generation tools generate better data for `denoise (profiled)` (since in release log for dt 3.0 it was mentioned that "old" way used 2 params, new uses 3, but old data is still useful because 3d param is interpolated from 1st 2 params)?

**Comment by rawfiner** [comment]
There should not be a noticeable difference if the shots were blurry enough, and with values all accross the tone range. Andreas' article gives a good method to have these conditions :-)
As for your last question, the noiseprofiling tool has not changed. It still generates 2 params, and denoise profiled guesses the third one. Before this guessing strategy was implemented, I tried to update the noise profiling tool to find the 3 parameters, but it was hard to make profiling work reliably enough, and that would have implied to redo all noise profiled so I finally chose to go for the guessing strategy, and give the user the ability to change the third parameter in the GUI if the guessed value does not fit its need.
